modules/tinymce/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Search and replace plugin would incorrectly find matching text inside non-editable root elements. #TINY-10162
 - Search and replace plugin would incorrectly find matching text inside SVG elements. #TINY-10162
 - Removed use of `async` for editor rendering which caused visual blinking when reloading the editor in-place. #TINY-10249
+- Toggling the formatting on the caret at the start of the editor, which initially has multiple styles applied, results in the incorrect removal of other formatting styles. #TINY-10132
```

Comment:
```suggestion
- Toggling off one format on the caret when multiple formats was toggled on would toggle all of them off. #TINY-10132
```

This doesn't only affect start of the editor. It can be end of a paragraph beginning of a word.

All these have this behavior all of them seems covered by you fix:
```
<p>foo bar|</p>
<p>foo| bar</p>
<p>|</p>
```